src/plots/mapbox/mapbox.js

```diff
@@ -722,11 +722,14 @@ proto.project = function(v) {
 proto.getView = function() {
     var map = this.map;
     var mapCenter = map.getCenter();
-    var center = { lon: mapCenter.lng, lat: mapCenter.lat };
+    var lon = mapCenter.lng;
+    var lat = mapCenter.lat;
+    var center = { lon: lon, lat: lat };
 
     var canvas = map.getCanvas();
-    var w = canvas.width;
-    var h = canvas.height;
+    var w = parseInt(canvas.style.width);
+    var h = parseInt(canvas.style.height);
```

Comment:
When you make a canvas, the way to give it correct resolution for a retina display is to set the style `width` and `height` to the css pixel size, and the `width` and `height` attributes to that size times `window.devicePixelRatio` - so we just needed to switch to style.